nextstrain/auspice
# Adds toggle for transmission lines

### Description of proposed changes    
Adds toggle for transmission lines.

@jameshadfield not exactly sure if I've got the translation stuff set up right, but let me know how this looks so far

### Related issue(s)  
Fixes #1095 
Related to #1095 

### Testing
1. Load up the app and locate the "Map Options" section in the side bar
1. Verify that there is a toggle switch that says "Show transmission lines"
1. Verify that toggling the switch will remove the transmission lines from the map UI 

**Tests**
- Didn't write any new tests, but updated the snapshot tests as the UI change broke the existing tests

### Thank you for contributing to Nextstrain!
